### PR TITLE
randomPercent to the RandomCode

### PR DESCRIPTION
**Pull request by winsvega**
randomPercent() in RandomCode

### CODE REVIEW

**Comment by axic** [comment]
Why not just add a `max` parameter to `randomUintGen`?

**Comment by winsvega** [comment]
that is what we are trying to avoid. before it was working with randomUintGen(maxValue)

File: test/tools/fuzzTesting/fuzzHelper.cpp [hunk 1]
```diff
@@ -83,12 +83,14 @@ namespace test
 {
 
 boost::random::mt19937 RandomCode::gen;
+boostIntDistrib RandomCode::prcentDist = boostIntDistrib (0, 100);
```
Comment (pirapira) [inline test/tools/fuzzTesting/fuzzHelper.cpp]: `percentDist`.

File: test/tools/fuzzTesting/fuzzHelper.cpp [hunk 2]
```diff
@@ -83,12 +83,14 @@ namespace test
...
 boostIntDistrib RandomCode::opCodeDist = boostIntDistrib (0, 255);
 boostIntDistrib RandomCode::opLengDist = boostIntDistrib (1, 32);
 boostIntDistrib RandomCode::opMemrDist = boostIntDistrib (0, 10485760);
 boostIntDistrib RandomCode::uniIntDist = boostIntDistrib (0, 0x7fffffff);
 boostUint64 RandomCode::uInt64Dist = boostUint64 (0, std::numeric_limits<uint64_t>::max());
 
+boostIntGenerator RandomCode::randPrcentGen = boostIntGenerator(gen, prcentDist);
```
Comment (pirapira) [inline test/tools/fuzzTesting/fuzzHelper.cpp]: `randPercentGen`.  Somehow I sense this was done to keep the widths uniform, but it's anyway not always possible to keep the same number of characters.

**Pull request merged**